### PR TITLE
update misc asserts

### PR DESCRIPTION
**Pull request by SirRGB**
Fixes https://github.com/openandroidinstaller-dev/openandroidinstaller/issues/375, https://github.com/openandroidinstaller-dev/openandroidinstaller/issues/332 and https://github.com/openandroidinstaller-dev/openandroidinstaller/issues/308

### CODE REVIEW

File: openandroidinstaller/assets/configs/hero2lte.yaml [hunk 1]
```diff
@@ -2,16 +2,21 @@ metadata:
   maintainer: Tobias Sterbak (tsterbak)
   device_name: Samsung Galaxy S7 Edge
   is_ab_device: false
-  device_code: hero2lte 
+  device_code: hero2lte
   supported_device_codes:
-    - hero2lte 
+    - hero2lte
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/hero2lte.yaml:7]: Are you sure these are all the same devices with the same hardware? Samsung is notorious for even having different cpus for different codes sometimes...

File: openandroidinstaller/assets/configs/hltetmo.yaml [hunk 2]
```diff
@@ -2,9 +2,13 @@ metadata:
...
   is_ab_device: false
-  device_code: hltetmo 
+  device_code: hltetmo
   supported_device_codes:
-    - hltetmo 
+    - hltetmo
+    - hlte
+    - hltecan
+    - hlteusc
+    - hltevzw
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/hltetmo.yaml:11]: see above

File: openandroidinstaller/assets/configs/jfltexx.yaml [hunk 3]
```diff
@@ -7,6 +7,8 @@ metadata:
     - jfltexx
     - jflte
     - jfltetmo
+    - jfltecan
+    - jgedlte
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/jfltexx.yaml:11]: see above

File: openandroidinstaller/assets/configs/klte.yaml [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
   supported_device_codes:
-    - klte 
+    - klte
+    - klteacg
+    - kltecan
+    - kltetmo
+    - klteub
+    - klteusc
+    - kltevzw
+    - kltexx
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/klte.yaml:14]: see above

File: openandroidinstaller/assets/configs/on7xelte.yaml [hunk 5]
```diff
@@ -2,17 +2,23 @@ metadata:
...
-  device_code: on7xelte 
+  device_code: on7xelte
   supported_device_codes:
     - on7xelte
+    - on7xeltedd
+    - on7xeltekl
+    - on7xeltekk
+    - on7xelteks
+    - on7xelteub
+    - on7xeltezt
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/on7xelte.yaml:13]: see above

File: openandroidinstaller/assets/configs/serranoltexx.yaml [hunk 6]
```diff
@@ -6,13 +6,15 @@ metadata:
   supported_device_codes:
     - serranolte
     - serranoltexx
+    - serranoltebmc
+    - serranoltektt
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/serranoltexx.yaml:10]: see above

File: openandroidinstaller/assets/configs/zerofltexx.yaml [hunk 7]
```diff
@@ -2,16 +2,17 @@ metadata:
   maintainer: Tobias Sterbak (tsterbak)
   device_name: Samsung Galaxy S6
   is_ab_device: false
-  device_code: zerofltexx 
+  device_code: zerofltexx
   supported_device_codes:
-    - zerofltexx 
+    - zerofltexx
+    - zeroflte
```
Comment (tsterbak) [inline openandroidinstaller/assets/configs/zerofltexx.yaml:8]: see above

File: openandroidinstaller/assets/configs/hero2lte.yaml (hunk 1 above)
Comment (SirRGB) [reply openandroidinstaller/assets/configs/hero2lte.yaml:7]: Please check out the links of the respective commits.
The asserts basically mean that the lineage build can be flashed on all of these device codes.
If these models were diff socs, they wouldnt be in a single device tree.

Comment (tsterbak) [reply openandroidinstaller/assets/configs/hero2lte.yaml:7]: Ah great! Thank you! :)

**Pull request merged**